src/components/GroupModal.js

```diff
@@ -240,14 +244,24 @@ const GroupModal = ({ cubeID, canEdit, children, ...props }) => {
                 </ListGroup>
               </Row>
               <Row noGutters>
-                {Number.isFinite(totalPrice) && (
-                  <TextBadge name="Price" className="mt-2 mr-2">
-                    <Tooltip text="TCGPlayer Market Price">${Math.round(totalPrice).toLocaleString()}</Tooltip>
+                {Number.isFinite(totalPriceUsd) && (
+                  <TextBadge name="Price USD" className="mt-2 mr-2">
+                    <Tooltip text="TCGPlayer Market Price">${Math.round(totalPriceUsd).toLocaleString()}</Tooltip>
```

Comment:
Using `Math.round` here instead of `toFixed(2)` is correct?

Reply:
This approach gets commas in there for big numbers (or whatever your locale format is)